generics/helpers/utils.js

```diff
@@ -284,6 +284,17 @@ function arrayOfObjectToArrayOfObjectId(ids) {
   return ids.map(obj => obj._id);
 }
 
+/**
+  * array of object to array objectId
+  * @function
+  * @name arrayOfObjectToArrayOfObjectId
+  * @returns {Boolean} returns a array of objectIds in string format
+*/
+
+function arrayOfObjectToArrayOfObjectIdInString(ids) {
```

Comment:
@ankitpws do see if there is a better name

Reply:
Changed